modules/GephiSimilarityComputer/src/main/java/Controller/MyFileImporter.java

```diff
@@ -0,0 +1,262 @@
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package Controller;
```

Comment:
Can you refactor all your files to include java.net.clementlevallois or something before each package? This is automatically done by Netbeans when you rename the package.